uniflow/flow/transform/transform_huggingface_flow.py

```diff
@@ -41,3 +42,8 @@ def run(self, nodes: Sequence[Node]) -> Sequence[Node]:
             Sequence[Node]: Nodes after running.
         """
         return self._model_op(nodes)
+
+
+class TransformHuggingFaceFlow(HuggingFaceModelFlow):
+    TAG = TRANSFORM
+
```

Comment:
qq: why we need to TransformHuggingFaceFlow to inherent HuggingFaceModelFlow?

Reply:
Let's discuss whether we need to keep both classes?!